Framework/Kernel/test/PropertyManagerTest.h

```diff
@@ -90,12 +90,12 @@ class PropertyManagerTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(manager.propertyCount(), 3);
 
     const std::vector<Property *> &props = manager.getProperties();
-    for (auto iter = props.begin(); iter != props.end(); ++iter) {
-      Property *prop = *iter;
+    for (auto iter : props) {
```

Comment:
`prop`?

Reply:
Collides with the variable definition below.